src/components/RightDockedModal.js

```diff
@@ -20,25 +21,73 @@ const propTypes = {
     /* Onyx Props */
     // Url currently in view
     currentURL: PropTypes.string,
+
+    isModalShown: PropTypes.bool.isRequired,
```

Comment:
Each propTypes needs to have an inline comment above it to give context for how to use the prop